pkg/vault/test/authorizer/TimelockAuthorizerRoot.test.ts

```diff
@@ -128,45 +128,5 @@ describe('TimelockAuthorizer', () => {
       expect(await authorizer.canRevoke(GENERAL_PERMISSION_SPECIFIER, grantee, WHERE_1)).to.be.false;
       expect(await authorizer.canRevoke(GENERAL_PERMISSION_SPECIFIER, grantee, EVERYWHERE)).to.be.false;
     });
-
-    it('can have their global grant permissions revoked by an authorized address for any contract', async () => {
```

Comment:
Because there's nothing special about granters that'd let them do it for us to test this :shrug: If you hadn't seen the prior version of the code, you wouldn't think to write this test

Reply:
correct, but I've seen it :grimacing: 

My point is that we are coming from that version, and we could've made a mistake while doing so. But yes, if we went straight to the current state we'd never have written that test so it's also fine not to have it.